test/code/plugins/oms_common_test.rb

```diff
@@ -94,6 +100,148 @@ def teardown
+      Common.HostnameFilePath = nil
+    end
+
+    def test_is_hostname_compliant?
+      assert(Common.is_hostname_compliant?('onetwo'), "Common.is_hostname_compliant?('onetwo') should be true.")
+      assert(Common.is_hostname_compliant?('x'), "Common.is_hostname_compliant?('x') should be true.")
+      assert(Common.is_hostname_compliant?(      'xxx123456789012345678901234567890123456789012345678901234567890'),
+            "Common.is_hostname_compliant?(      'xxx123456789012345678901234567890123456789012345678901234567890') should be true.")
+
+      assert_false(Common.is_hostname_compliant?('xxx123456789012345678901234567890123456789012345678901234567890x'),
+                  "Common.is_hostname_compliant?('xxx123456789012345678901234567890123456789012345678901234567890x') should NOT be true.")
+      assert_false(Common.is_hostname_compliant?('one.two'), "Common.is_hostname_compliant?('one.two') should NOT be true.")
+      assert_false(Common.is_hostname_compliant?('one:two'), "Common.is_hostname_compliant?('one:two') should NOT be true.")
+      assert_false(Common.is_hostname_compliant?(nil), "Common.is_hostname_compliant?(nil) should NOT be true.")
+      assert_false(Common.is_hostname_compliant?(''), "Common.is_hostname_compliant?('') should NOT be true.")
+    end
+
+    def test_is_like_ipv4_string?
+      assert(Common.is_like_ipv4_string?('192.168.0.5'),     "Common.is_like_ipv4_string?('192.168.0.5') should be true.")
+      assert(Common.is_like_ipv4_string?('1.2.3.4'),         "Common.is_like_ipv4_string?('1.2.3.4') should be true.")
+      assert(Common.is_like_ipv4_string?('254.254.254.254'), "Common.is_like_ipv4_string?('254.254.254.254') should be true.")
+      assert(Common.is_like_ipv4_string?('1.0.0.0'),         "Common.is_like_ipv4_string?('1.0.0.0') should be true.")
+
+      assert_false(Common.is_like_ipv4_string?('1.0.0.444'),       "Common.is_like_ipv4_string?('1.0.0.444') should NOT be true.")
+      assert_false(Common.is_like_ipv4_string?('0.2.3.4'),         "Common.is_like_ipv4_string?('0.2.3.4') should NOT be true.")
+      assert_false(Common.is_like_ipv4_string?('1.2.3.255'),       "Common.is_like_ipv4_string?('1.2.3.255') should NOT be true.")
+      assert_false(Common.is_like_ipv4_string?('255.255.255.255'), "Common.is_like_ipv4_string?('1.2.3.255') should NOT be true.")
+      assert_false(Common.is_like_ipv4_string?('192.168.0'),       "Common.is_like_ipv4_string?('192.168.0') should NOT be true.")
+      assert_false(Common.is_like_ipv4_string?('192.168..5'),      "Common.is_like_ipv4_string?('192.168..5') should NOT be true.")
+      assert_false(Common.is_like_ipv4_string?('192.168.0.5.5'),   "Common.is_like_ipv4_string?('192.168.0.5.5') should NOT be true.")
+      assert_false(Common.is_like_ipv4_string?('192.168.0.5.A'),   "Common.is_like_ipv4_string?('192.168.0.5.5') should NOT be true.")
+    end
+
+    def test_is_like_ipv6_string?
```

Comment:
I prefer the layout of this method to the layout of `test_is_like_ipv4_string?` or `test_is_hostname_compliant?`. Can you update those test methods to use a block with constants to check, and then a block of assert statements? It looks very clear that way.